app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationCollection/integrationCollection.jsx

```diff
@@ -23,12 +23,10 @@ import styles from './integrationCollection.scss';
 const cx = classNames.bind(styles);
 
 export const IntegrationCollection = ({ items, disabled, openIntegration }) => {
-  const sortedItems = items.sort((a, b) => b.creationDate - a.creationDate);
-
   return (
```

Comment:
Optional:
I think here we can delete `return` statement.

Reply:
Done.